src/map.react.js

```diff
@@ -278,10 +282,6 @@ export default class MapGL extends Component {
     return 'inherit';
   }
 
-  _getMap() {
```

Comment:
Yep that sounds good. We can move towards removing `_getMap()` eventually as well but we should add deprecation warnings for a couple of versions so people aren't caught completely off guard.